packages/wrangler/src/index.tsx

```diff
@@ -333,6 +334,9 @@ export async function main(argv: string[]): Promise<void> {
         }
       }
 
+      const packageJsonContent = JSON.parse(
+        await readFile(pathToPackageJson, "utf-8")
+      );
```

Comment:
How about doing something like this here:

```ts
const shouldWritePackageJsonScripts =
    !packageJsonContent.scripts?.start &&
    !packageJsonContent.scripts?.deploy &&
    shouldCreatePackageJson;
```

Then later on you can just have 

```ts
if (shouldWritePackageJsonScripts) {
  ...
}
```

Reply:
I agree. Slight abstraction and more explicit what it is checking.